CHANGES.rst

```diff
@@ -5,4 +5,5 @@ Change log for gocept.template_rewrite
 0.1 (unreleased)
 ================
 
-initial release
+- File discovery now also includes TrustedFSPageTemplate files, which
+are registered with the file ending .xpt.
```

Comment:
ReST expects the second line indented:
```suggestion
  are registered with the file ending `.xpt`.
```

Reply:
Thank you for the review. I just installed a new rst-plugin for my editor - I still have not found a real good one.